test/ranges.jl

```diff
@@ -710,13 +710,13 @@ function test_linspace_identity(r::Range{T}, mr) where T
     @test -collect(r) == collect(mr)
     @test isa(-r, typeof(r))
 
-    @test 1 + r + (-1) == r
-    @test 1 + collect(r) == collect(1 + r) == collect(r + 1)
-    @test isa(1 + r + (-1), typeof(r))
-    @test 1 - r - 1 == mr
-    @test 1 - collect(r) == collect(1 - r) == collect(1 + mr)
-    @test collect(r) - 1 == collect(r - 1) == -collect(mr + 1)
-    @test isa(1 - r - 1, typeof(r))
+    @test 1 .+ r .+ (-1) == r
+    @test 1 .+ collect(r) == collect(1 .+ r) == collect(r .+ 1)
+    @test_broken isa(1 .+ r .+ (-1), typeof(r))
```

Comment:
Right. I've been playing with this to make the test better... I just realized there was no `1 .+ r` specialization either, for ranges. (In fact, there's a couple other strange definitions there I will deal with, like `3 / (1:3)` still exists whereas `3 / [1,2,3]` has already been removed in a previous vectorized operations purge).